data/pages-content/sponsor/information.md

```diff
@@ -203,32 +206,27 @@ distribute them at your booth. We strongly encourage you to distribute gifts
 made of environmentally friendly materials, to focus on quality rather than
 quantity to reduce waste.
 
-Shipping and customs for your goods are handled by
-[DB Schenker](https://www.dbschenker.com/cz-en).
-
-**Contacts**:
+Shipping your goods to the venue is exclusively handled by
+[DB Schenker](https://www.dbschenker.com/cz-en). Please get in touch with Martin ULIČNÝ from DB Schenker directly to discuss costs and shipping arrangement.
 
+**Contact**:
 
-- Martin ULIČNÝ - martin.+420 242 405 160, +TBD (mobile)
+- Martin ULIČNÝ
+  - Tel: +420 242 405 160
+  - E-mail: martin.ulicny@dbschenker.com
 
 Here are shipping information provided by DB Schenker:
 
-- [Shipping Guidelines](TBD).
-  **Important: Please provide DB Schenker your Company name, point of contact name
-  and email.** Contact DB Schenker for Details.
+- [Shipping Guidelines & Address (.docx)](https://docs.google.com/document/d/1tgDfBikrFo1e9rtWWCNgNbe8q2I8mELl/edit).
```

Comment:
Maybe we should "Copy link" instead, so that people can view the document, but not edit?

Reply:
yup, the doc's permission was set to view only but /view would be more accurate indeed!